docs/v3.x/deployment/ibm-cloud.md

```diff
@@ -0,0 +1,332 @@
+        host: env('DATABASE_HOST', '127.0.0.1'),
+        port: env.int('DATABASE_PORT', 27017),
+        database: env('DATABASE_NAME', 'strapi'),
+        username: env('DATABASE_USERNAME', ''),
+        password: env('DATABASE_PASSWORD', ''),
+      },
+      options: {
+        ssl: true,
+      },
+    },
+  },
+});
+```
+
+#### 5. Install the `pg` node module
+
+Unless you originally installed Strapi with PostgreSQL, you need to install the [pg](https://www.npmjs.com/package/pg) node module.
+
+`Path: ./my-project/`
+
+```bash
+npm install pg --save
+```
+
+:::
+
+::: tab MongoDB
+
+### IBM MongoDB
+
+(Using Strapi and MongoDB requires different set-up and different configuration steps. You cannot use `--quickstart` to develop a `MongoDB` Strapi project.)
+
+Please follow these steps to **deploy a Strapi app with MongoDB on IBM Cloud**.
+
+#### 1. Create a [IBM Cloud MongoDB service](https://cloud.ibm.com/catalog/services/databases-for-mongodb) for using MongoDB
+
+The easiest way is to do it through the web management panel.
+
+#### 2. Retrieve database credentials
+
+Once the new service is ready, new credentials need to be created (`service credentials` section).
+
+A huge JSON file is generated but we are interested in the field called `connection.mongodb.composed`. There you can find the values you need for the next step.
+
+```txt
+mongodb://ibm_cloud_f999fe77_d44c_44f1_ad9a_d950111f381b:c6a1c55197f4855eb47d03401ddc5bfa9c8ba76472f420d7a1f968@bbccc1c1-6013-40c1-b17b-8b45ff36a96f-0.8117127f814b4a6ea643610826cd2123.databases.appdomain.cloud:32083,cacab8c8-7029-40c1-b17b-8b45ff36a96f-1.8117147f814b4b2ea643610826cd2046.databases.appdomain.cloud:32083/ibmclouddb?authSource=admin&replicaSet=replset
+```
+
+(This url is read like so: `mongodb:// USERNAME : PASSWORD @ HOST : PORT / DATABASE_NAME`)
+
+#### 3. Set environment variables
+
+Strapi also supports a URI format, instead a variable for each database connection configuration (host, username, etc.). So, you only have to set one environment variables in the IBM Cloud config:
+
+```bash
+ibmcloud app env-set "My Project" DATABASE_URI "mongodb://ibm_cloud_f999fe77_d44c_44f1_ad9a_d950111f381b:c6a1c55197f4855eb47d03401ddc5bfa9c8ba76472f420d7a1f968@bbccc1c1-6013-40c1-b17b-8b45ff36a96f-0.8117127f814b4a6ea643610826cd2123.databases.appdomain.cloud:32083,cacab8c8-7029-40c1-b17b-8b45ff36a96f-1.8117147f814b4b2ea643610826cd2046.databases.appdomain.cloud:32083/ibmclouddb?authSource=admin&replicaSet=replset"
+```
+
+Please replace these above values with your actual values.
+
+#### 4. Update your database config file
+
+`Path: ./config/database.js`.
+
+```js
+module.exports = ({ env }) => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'mongoose',
+      settings: {
+        uri: env('DATABASE_URI', 'mongodb://localhost:27017/myproject'),
+      },
+      options: {
+        ssl: true,
+      },
+    },
+  },
+});
+```
+
+:::
+
+::::
+
+#### 9. Deploy changes
```

Comment:
I'm not seeing anything relating to configuring the proxy but earlier on the app yaml file referenced enforcing HTTPs. Depending on how CF is proxying stuff the users may need to configure the proxy for proper file upload references or any authentication providers. See: https://strapi.io/documentation/v3.x/concepts/configurations.html#server

Reply:
Sure, my fault.  I forget to add the changes in server.js.

Adding it, sorry.

Reply:
Np :)

Reply:
Added here: https://github.com/jesusprubio/strapi/blob/deploy-ibm/docs/v3.x/deployment/ibm-cloud.md#7-ibm-cloud-proxy-set-up